src/scss/constants.scss

```diff
@@ -66,7 +66,7 @@ $send-msg-online-color: #78b553;
 
 $map-sidebar-width: 20%;
 $chat-sidebar-width: 360px;
-$navbar-height: 66px;
+$navbar-height: 54px;
```

Comment:
There are components (mostly admin related, possibly chat) that don't use `$navbar-height` but hardcoded `66px`. This navbar height change may cause minor issues with those. Did project wide search reveal more places than `Admin.scss`?

Reply:
fixed 61d9df7, created separate var